Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1304,6 +1304,11 @@ void AXObjectCache::handleChildrenChanged(AccessibilityObject& object)
     else if (auto* axRow = dynamicDowncast<AccessibilityTableRow>(object)) {
         if (auto* parentTable = axRow->parentTable())
             deferRecomputeTableCellSlots(*parentTable);
+    } else if (auto* scrollView = dynamicDowncast<AccessibilityScrollView>(object)) {
+        if (auto* webarea = scrollView->webAreaObject()) {
+            webarea->setNeedsToUpdateSubtree();
+            webarea->setNeedsToUpdateChildren();
+        }
```

Comment:
Can we narrow this to only being applicable to iframe scrollviews / web areas?

I think we've standardized around capitalizing webarea like webArea.

Do we need to set both `setNeedsToUpdateSubtree` and `setNeedsToUpdateChildren` to fix the bug? Can we add a comment explaining we need to propagate this flag / flags down when iframes change attributes that affect the accessibility tree?

Reply:
auto* webarea --> RefPtr since we are using this pointer after some non-trivial operation.

Reply:
> Do we need to set both setNeedsToUpdateSubtree and setNeedsToUpdateChildren to fix the bug? Can we add a comment explaining we need to propagate this flag / flags down when iframes change attributes that affect the accessibility tree?

Just verified that we do. This mimics the behavior of `handleChildrenChanged` beneath the bail out condition that ScrollViews hit—both setNeedsToUpdateSubtree and setNeedsToUpdateChildren are called on `object`